.codeclimate.yml

```diff
@@ -2,6 +2,9 @@
 engines:
   shellcheck:
     enabled: true
+    checks:
+      SC2154:
+        enabled: false
```

Comment:
This check is very much needed. Please don't disable it. Follow the advice of all upper cases for exported variables like renaming noOfErrors to "NO_OF_ERRORS".